README.md

```diff
@@ -61,6 +61,11 @@ brew install font-monaspace
 ### Windows
 You can manually drag the fonts from the `fonts/otf` or `fonts/variable` directory into `C:\Windows\Fonts`. Alternatively, right-click the fonts you want and click Install.
 
+```powershell
+$ cd util
+$ ./install_windows.sh
```

Comment:
./install_windows.cmd  instead of *.sh

Reply:
Correction done, code update pushed.

Reply:
I used the script, works great! Hopefully the Monaspace team accepts the PR!